bake/compose.go

```diff
@@ -44,6 +46,15 @@ func ParseCompose(dt []byte) (*Config, error) {
 				dockerfilePath := s.Build.Dockerfile
 				dockerfilePathP = &dockerfilePath
 			}
+			// Check if there's actually a dockerfile mentioned
+			if dockerfilePathP == nil && contextPathP == nil {
+				// if not make sure they're setting an image or it's invalid d-c.yml
+				if s.Image == "" {
+					return nil, fmt.Errorf("Compose file invalid: Service %s has neither an image nor a build context specified. At least one must be provided.", s.Name)
```

Comment:
`errors.Errorf` and start with a lowercase letter

Reply:
actually, should this be invalid? why not just do default build? I agree that we should skip if image is set and there is no build block. and instead of checking a context I think we should check the `s.Build` against the zero value.

Reply:
It is invalid if you try and validate a broken service definition (like hack/demo-env/examples/compose/docker-compose.yml) w/ docker-compose it spits out that error.

Reply:
Ok, does it error every time there is a build block without `context: .`? Is it in the spec? It obviously seems to be supported by the go parser used by stacks etc. Maybe we should patch the python side?